src/constants/message-types.js

```diff
@@ -0,0 +1,2 @@
+export const COMMS_EVENT = 'commsMessage'
+export const FILE_METADATA = 'metadata'
```

Comment:
General question: Should we exclude metadata logic from this repo since its not being used and its likely to be different on CDP?

Reply:
discussed and agreed to leave it